issue_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+Thanks for notifying us of a new issue!
+
+Before you submit, please make sure you do the following:
```

Comment:
"please make sure you have done the following" sounds more natural to me, but I'm not sure if it really is (I'm not a native English speaker :-)).